vmcnet/train/runners.py

```diff
@@ -277,11 +281,17 @@ def _get_clipping_fn(
 
 def _get_energy_fns(
     vmc_config: ConfigDict,
+    problem_config: ConfigDict,
```

Comment:
Do you think it would be beneficial to have a softening_config that includes ee and ei, or is using problem sufficient?

Reply:
I think it's fine in problem, the potential is really part of the specification of the problem just like the ion locations and charges.